src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylePairs = sourceString
+    .trim()
+    .split(';')
+    .map((pair) => pair.trim().split(':'))
+    .filter((pair) => pair.length > 1);
```

Comment:
Instead of filtering for pairs with length greater than 1, it would be more accurate to filter out empty strings to avoid creating invalid entries in the object.